cmd/plural/init.go

```diff
@@ -43,9 +43,14 @@ func handleInit(c *cli.Context) error {
 	if err != nil {
 		return err
 	}
-	defer func(prov provider.Provider) {
+	kv, err := crypto.GenerateKeyValidator()
```

Comment:
we should also run this in the setup phase of the cloud shell, and also have an independent crypto command to generate it manually for old repos (i'll maybe call that in the console or something)

Reply:
@zreigz It's in `pkg/server/setup.go`